Buildings/Fluid/Air/BaseClasses/Examples/ElectricHeater.mo

```diff
@@ -0,0 +1,49 @@
+within Buildings.Fluid.Air.BaseClasses.Examples;
+model ElectricHeater "Test model for ElectricHeater"
+  extends Modelica.Icons.Example;
+  extends Buildings.Fluid.HeatExchangers.Examples.BaseClasses.Heater(
+    redeclare package Medium = Buildings.Media.Air,
+    m_flow_nominal=V*1.2*6/3600,
+    Q_flow_nominal=30*6*6,
+    mov(nominalValuesDefineDefaultPressureCurve=true, dp_nominal=1200));
+  Buildings.Fluid.Air.BaseClasses.ElectricHeater eleHea(
+    redeclare package Medium = Medium,
+    m_flow_nominal=m_flow_nominal,
+    QMax_flow=Q_flow_nominal,
+    eff=0.8,
```

Comment:
You're right. The efficiency should be 1. I add this parameter because EnergyPlus: ElectricAirHeatingCoil does the same thing.